content/version/2/0/code_of_conduct.md

```diff
@@ -11,15 +11,19 @@ We as members, contributors, and leaders pledge to make participation in our com
 
 We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
 
+We believe that a diverse community where people treat each other with respect is stronger, more vibrant, and has more potential contributors and more sources for ideas. We aim for more diversity.
+
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our community include:
 
 * Demonstrating empathy and kindness toward other people
+* Demonstrating mutual respect and tolerance towards other people
 * Being respectful of differing opinions, viewpoints, and experiences
 * Giving and gracefully accepting constructive feedback
 * Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
 * Focusing on what is best not just for us as individuals, but for the overall community
+* Using inclusive, gender-neutral language in the documents we provide and when we talk to people. When referring to a group of people, we aim to use gender-neutral terms like “team”, “folks”, “everyone”.
```

Comment:
The language here feels a bit casual. Would you like to take a pass at making it match more of the rest of the style of the document, or would you rather that I propose new language?

Reply:
Made some changes inline. Take a look?

Reply:
What Language @CoralineAda ? Kindly inform.